test/e2e/e2e_suite_test.go

```diff
@@ -25,8 +25,8 @@ import (
 	"testing"
 	"time"
 
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
+	"github.com/onsi/ginkgo/v2"
```

Comment:
As we discussed let's keep the "import dot" for ginkgo and gomega to align with the convention.

But if you there are similar situation for other module IMO we should fix it if it causes confusion.

Reply:
https://github.com/onsi/composition-ginkgo-example

Reply:
Modified.